Added Feather Icons Support

pablo-lp opened the pull request.

pablo-lp commented:
Feather Icons Example Config

```
icons: {
	type: 'feather',
	time: 'clock',
	date: 'calendar',
	up: 'arrow-up',
	down: 'arrow-down',
	previous: 'arrow-left',
	next: 'arrow-right'
}
```

tonix-tuft commented:
This project seems to be unmaintained as the maintainer is unresponsive.
I have sent several pull requests, but they all have been ignored.

I have some plans about developing it further, so I have forked both
`tempusdominus/core` and `tempusdominus/bootstrap-4`:

- https://github.com/tonix-tuft/tempusdominus/tree/fork - Tempus Dominus Core (fork of tempusdominus/core)
- https://github.com/tonix-tuft/tempusdominus-bootstrap/tree/fork - Tempus Dominus Bootstrap (fork of tempusdominus/bootstrap-4)

They are also available through NPM:

- http://npmjs.com/tempusdominus
- https://www.npmjs.com/package/tempusdominus-bootstrap

In these two forks (I worked on the `fork` branch on both, not on `master`) I have updated all the NPM dependencies and build scripts,
added some new features and fixed some bugs I have discovered while using Tempus Dominus.

For now, I have left the `master` branch in my forks intact and the same as in the original repos, as I was waiting for my PR being merged
in the original repos and I am adding patches/bugfixes and new features on the `fork` branch in both forked projects.

If you have some pending PR you would like to merge, I invite you to fork my forks instead and we can work
on it together in the free time.

@HaroldPutman already sent me a PR which got merged in tonix-tuft/tempusdominus -> https://github.com/tonix-tuft/tempusdominus/pull/1.

P.S.: I have also created a React version of Tempus Dominus Bootstrap, available here (I hope you enjoy it 😃):

- https://github.com/tonix-tuft/react-tempusdominus-bootstrap - GitHub Repo
- https://tonix-tuft.github.io/react-tempusdominus-bootstrap/ - Demo

pablo-lp commented:
Thanks, I made a PR to your fork

> El 20 may 2020, a las 14:31, Anton Bagdatyev (Tonix) <notifications@github.com> escribió:
> 
> This project seems to be unmaintained as the maintainer is unresponsive.
> I have sent several pull requests, but they all have been ignored.
> 
> I have some plans about developing it further, so I have forked both
> tempusdominus/core and tempusdominus/bootstrap-4:
> 
> https://github.com/tonix-tuft/tempusdominus/tree/fork - Tempus Dominus Core (fork of tempusdominus/core)
> https://github.com/tonix-tuft/tempusdominus-bootstrap/tree/fork - Tempus Dominus Bootstrap (fork of tempusdominus/bootstrap-4)
> They are also available through NPM:
> 
> http://npmjs.com/tempusdominus
> https://www.npmjs.com/package/tempusdominus-bootstrap
> In these two forks (I worked on the fork branch on both, not on master) I have updated all the NPM dependencies and build scripts,
> added some new features and fixed some bugs I have discovered while using Tempus Dominus.
> 
> For now, I have left the master branch in my forks intact and the same as in the original repos, as I was waiting for my PR being merged
> in the original repos and I am adding patches/bugfixes and new features on the fork branch in both forked projects.
> 
> If you have some pending PR you would like to merge, I invite you to fork my forks instead and we can work
> on it together in the free time.
> 
> @HaroldPutman already sent me a PR which got merged in tonix-tuft/tempusdominus -> tonix-tuft/tempusdominus#1.
> 
> P.S.: I have also created a React version of Tempus Dominus Bootstrap, available here (I hope you enjoy it 😃):
> 
> https://github.com/tonix-tuft/react-tempusdominus-bootstrap - GitHub Repo
> https://tonix-tuft.github.io/react-tempusdominus-bootstrap/ - Demo
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

tonix-tuft commented:
Hello guys,

I have sent all the PRs we have already merged into my two forks (https://github.com/tonix-tuft/tempusdominus and https://github.com/tonix-tuft/tempusdominus-bootstrap)
to the main repo of Eonasdan directly on `master`.

I wanna thank @HaroldPutman, @pablo-lp and @zondaTW for their contribution.

I think that now it makes sense for us to continue to work on this project in my two forks and from the `master` branch instead of the `fork` branch I have created and used so far
because I was waiting for @Eonasdan to merge the features we added to its main repo.

But again, as @Eonasdan was and still is unresponsive, I think it would make sense to continue the development from the two forked repos https://github.com/tonix-tuft/tempusdominus and https://github.com/tonix-tuft/tempusdominus-bootstrap now using the `master` branch instead of `fork` as the starting point from which to fork the branches for your PRs.

As always, PRs for new features, bugfixes are welcome! Thank you. Stay tuned.

Eonasdan commented:
closed via #360
